metricbeat/module/elasticsearch/pending_tasks/data.go

```diff
@@ -29,28 +31,34 @@ var (
 	schema = s.Schema{
 		"insert_order":     c.Int("insert_order"),
 		"priority":         c.Str("priority"),
-		"source":           c.Str("source"),
+		"source":           c.Str("source", s.Required),
```

Comment:
How did you pick this field as required?

Reply:
There is a test failing if this field is not present, but I'm thinking now that all fields here seem to be required, I'm going to revert this part.